ci-operator/step-registry/ipi/azure/ovn/hybrid/ipi-azure-ovn-hybrid-workflow.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "ipi/azure/ovn/hybrid/ipi-azure-ovn-hybrid-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		],
+		"reviewers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		]
+	}
+}
```

Comment:
Please add newline

Reply:
This is a generated file, its supposed to not have a newline